### PR TITLE
Add AppVeyor Windows builds and releases

### LINKED ISSUE #204: Prebuilt Windows binaries
Getting a Windows binary is a PITA. It would be quite useful to build and publish a Windows exe using Appveyor.

(Would also be good for C++ Jsonnet: google/jsonnet#469)

### PR DESCRIPTION
**Pull request by marcelocantos**
Fixes #204

This PR cannot be approved in its current state (see comments). It is primarily for review and discussion.

Current output at https://github.com/marcelocantos/go-jsonnet/releases

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+build_script:
+  - go get .\jsonnet
+  - go build -o jsonnet\jsonnet.exe .\jsonnet
+
+artifacts:
+  - path: jsonnet/jsonnet.exe
+    name: exe
+
+deploy:
```
[the remark this answers is not in the document]
Comment (sbarzowski) [reply appveyor.yml]: I've added an issue for that: #207.

**Comment by sparkprime** [comment]
Let's get the filename fixes in first

**Comment by sparkprime** [comment]
I think you can just branch this, rebase to kill the appveyor config, and have a totally finished & non-controversial PR.

**Comment by sparkprime** [comment]
Thanks!  Appveyor builds would still be nice, let's talk about it

**Pull request merged**